### DISCUSSION TITLE
Small ideas to help Vitebook grow

### DISCUSSION THREAD

**Discussion by tillsanders** [vitebook/vitebook#58] [General]
I just discovered Vitebook and I see great potential for it. Coming from Storybook I like the simplicity of the toolchain (obviously). I would love to see this project grow, so why don't we come up with some suggestions to set help set this up for the long run!

Currently, the website and live examples are fantastic! What I was missing was a way to **follow this project**. I can watch the repo, but I think many people would like something like a more curated news channel. A twitter account maybe or an email newsletter. I want to be in the loop for the big developments.

Also, I would like to **show my support with maybe a small donation**. Something like Buy Me A Coffee, Ko-fi, GitHub Sponsors or OpenCollective. It will probably not collect that much in the beginning, but it shows the love and support of the community and that builds trust. As a developer I need to rely on the future of the project.

Looking forward to your thoughts!
Cheers,
Till

**Comment by gyurielf** [comment]
@tillsanders  I agree with you!
Maybe a Discord server is a good idea too.

**Reply by tillsanders** [reply]
Personally, I'm not a big fan of all these communities moving to Discord because you're keeping a lot of valuable information private. Search engines don't find it there.

But yeah, it would be nice to get more conversation going. There's already much conversation in the issues, like the open feedback thread. That's a good start 👍
